pkgs/development/python-modules/slack-sdk/default.nix

```diff
@@ -29,7 +29,7 @@ buildPythonPackage rec {
     owner = "slackapi";
     repo = "python-slack-sdk";
     rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
This was added by @r-ryantm. I guess that `refs/tags/` would be back with the next update.